sda-commons-client-jersey/src/test/java/org/sdase/commons/client/jersey/ClientProxyTest.java

```diff
@@ -34,13 +33,11 @@
   private static final WireMockClassRule PROXY_WIRE =
       new WireMockClassRule(wireMockConfig().dynamicPort());
 
-  private static final LazyRule<SystemPropertyRule> PROP =
-      new LazyRule<>(
-          () ->
-              new SystemPropertyRule()
-                  .setProperty("http.proxyHost", "localhost")
-                  .setProperty("http.proxyPort", "" + PROXY_WIRE.port())
-                  .setProperty("http.nonProxyHosts", "localhost"));
+  private static final SystemPropertyRule PROP =
+      new SystemPropertyRule()
+          .setProperty("http.proxyHost", "localhost")
+          .setProperty("http.proxyPort", () -> "" + PROXY_WIRE.port())
```

Comment:
Can we extend this to pass `Object` instead of `String`. Later call `.toString()`.

Reply:
This is the Dropwizard style and we have the same problems there: `public static ConfigOverride config(String key, Supplier<String> value) {`. But we could change it to Object.

Reply:
Ah ok, thought that Dropwizard is using an object here. No keep it like Dropwizard.